programs/test/benchmark.c

```diff
@@ -158,7 +168,8 @@ do {                                                                    \
 
 #define MEMORY_MEASURE_PRINT( title_len )                               \
     mbedtls_memory_buffer_alloc_max_get( &max_used, &max_blocks );      \
-    for( ii = 12 - (title_len); ii != 0; ii-- ) mbedtls_printf( " " );  \
+    ii = TITLE_SPACE > (title_len) ? TITLE_SPACE - (title_len) : 1;     \
+    while( ii-- ) mbedtls_printf( " " );                                \
```

Comment:
Preexisting: why not use `printf( "%-16s", … )`, or now `printf( "%-" #TITLE_SPACE "s", … )`?

Reply:
You'd need to use that when printing the title I think (after is too late), and then when `MEMORY_DEBUG` is disabled, you'd have trailing spaces in the output - or you'd have to make this conditional on `MEMORY_DEBUG` but since printing the title is made inside the body of a macro, prepocessor conditionals are not very convenient...